pygdf/tests/test_gpu_arrow_parser.py

```diff
@@ -1,13 +1,81 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
-
+import pytest
 import logging
 
+try:
+    import pyarrow as pa
+    arrow_version = pa.__version__
+except ImportError as msg:
+    print('Failed to import pyarrow: {}'.format(msg))
+    pa = None
+    arrow_version = None
+
 import numpy as np
 from numba import cuda
 
 from pygdf.gpuarrow import GpuArrowReader
 
 
+def make_gpu_parse_arrow_data_batch():
+    lat = np.array([39.297604, 41.785984, 29.533695, 41.785984,
+                    39.297604, 39.1754, 29.645418, 36.124477,
+                    29.533695, 32.847115, 29.533695, 32.847115,
+                    41.785984, 38.17439, 40.491467, 41.785984,
+                    33.66364, 30.194532, 41.785984, 36.124477,
+                    28.428888, 41.785984, 29.533695],
+                   dtype=np.float32)
+    lon = np.array([-94.713905, -87.752426, -98.46978, -87.752426,
+                    -94.713905, -76.6682, -95.27889, -86.678185,
+                    -98.46978, -96.85177,
+                    -98.46978, -96.85177, -87.752426, -85.736,
+                    -80.23287, -87.752426,
+                    -101.82278, -97.66987, -87.752426, -86.678185, -81.316025,
+                    -87.752426, -98.46978], dtype=np.float32)
```

Comment:
Could we generate this data using a static seed instead of hardcoding it in?

Reply:
Sure, I'll do it.

Reply:
Done.